src/components/CharacterAssignment.jsx

```diff
@@ -28,26 +35,10 @@ const CharacterAssignment = ({ actors, currentScriptId }) => {
     updateActorColumns(actors);
   }, [actors]);
 
-  /**
-   * Column constructor
-   * @param {string} id id used for drag and drop reference
-   * @param {string} title Column title that will be displayed
-   * @param {object} characters Contains Character objects indexed by their id
-   * @constructor
-   */
-  function Column(id, title, characters) {
-    this.id = id;
-    this.title = title;
-    this.characterIds = [];
-    this.getLineCount = function() {
-      if (characters === null) return;
-      return this.characterIds.reduce((acc, curr) => acc + characters[curr].lineCount, 0);
-    };
-  }
 
-  /**
-   * Create the actor columns and populate them with the assigned characters.
-   * Update state with the new columns, columnOrder, and characters
+  /** @function setupColumns
```

Comment:
Is this to handle the arrow function not being detected by jsdoc? If so, an alternate solution could be to switch these to using a regular function:
```js
const setupColumns = function(currentScriptId, actors) {
// or probably also just this, though I forget what exactly the implications are between const A = function() and function A() so it might not be right
function setupColumns(currentScriptId, actors) {
```